core/types/liquidity.go

```diff
@@ -405,14 +405,13 @@ type LiquidityMonitoringParameters struct {
 }
 
 func (l LiquidityMonitoringParameters) IntoProto() *proto.LiquidityMonitoringParameters {
-	tr, _ := l.TriggeringRatio.Float64()
 	var params *proto.TargetStakeParameters
 	if l.TargetStakeParameters != nil {
 		params = l.TargetStakeParameters.IntoProto()
 	}
 	return &proto.LiquidityMonitoringParameters{
 		TargetStakeParameters: params,
-		TriggeringRatio:       tr,
+		TriggeringRatio:       l.TriggeringRatio.String(),
```

Comment:
Can `l.TriggeringRatio` ever be `nil`?

Reply:
No, it is `decimal.Decimal`, not a pointer to it.